docs/ProductSummarySKUName.md

```diff
@@ -0,0 +1,58 @@
+# Product Summary Name
+
+## Description
+
+`ProductSummarySKUName` is a VTEX Component that renders the selected SKU's name.
+This Component can be imported and used by any VTEX App.
+
+:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
+
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+  - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+
+## Usage
```

Comment:
This is the old README model. 

You can follow this instead:
https://github.com/vtex-apps/breadcrumb

Reply:
Done, please check